src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Aggregator/FunctionResultLog.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal class FunctionResultLog
```

Comment:
Upon further inspection, doing this will require me to add a new interface like `IAggregator : IAsyncCollector<FunctionInstanceLogEntry>`, with no new members. That would let me continue to differentiate between the `FastLogger` and the `Aggregator`. Otherwise, you can only have one that matches that interface registered with the `ServiceProvider`. It's less code, so maybe it's better that way -- and it does make it easier to find all your aggregators since there's a specific interface.